### ISSUE TITLE
hardhat node broken with hardhat 2.0.9

### ISSUE THREAD

**Issue by gitpusha** [wighawag/hardhat-deploy#69]
```json
    "hardhat": "2.0.9",
    "hardhat-deploy": "0.7.0-beta.44",
 ```

I get the following error when running `npx hardhat node` : 

![image](https://user-images.githubusercontent.com/36712489/107624544-5a6f4f00-6c5b-11eb-998a-f1f67d6993dc.png)

**Comment by wighawag** [comment]
hmm, hardhat-deploy do not use `merkle-patricia-tree` by itself, it is a dependency of hardhat, 
I suspect the issue is elsewhere
if you have more info, like a reproducible repo, I could look into it more

**Comment by gitpusha** [comment]
Sorry have no repo to share for this. 

However, when I switch to hardhat version `2.0.8` it works fine again. 

So something off about the mix of `hardhat-deploy` and `hardhat 2.0.9` for the overriden `hardhat node` task (which attempts to deploy contracts automatically)

**Comment by wighawag** [comment]
@alcuadrado @fvictorio any idea ?
The latest changes on the node task do not seem to be related to the issue here : https://github.com/nomiclabs/hardhat/commit/8976e20822179553fabcb53a66056390aac788f6#diff-9b5fcdc22247b95bdf98fbb03e7c3476d32df3dee3243af01773083d20befb5e

maybe it is an issue elsewhere ?

@gitpusha I suppose deploying the contract via `hardhat deploy` (without --network argument) work ?

**Comment by wighawag** [comment]
Ok, as I guess, this is indeed unrelated to hardhat-deploy, see : https://github.com/nomiclabs/hardhat/issues/1237
@gitpusha try to use latest hardhat (2.0.10) or use a previous version of node as I assume you use node 14.

**Comment by fvictorio** [comment]
This was fixed yesterday on 2.0.10 :slightly_smiling_face:

**Comment by gitpusha** [comment]
@wighawag @fvictorio thanks for helping out!
